src/HealthGPS.Console/configuration.cpp

```diff
@@ -353,21 +335,12 @@ ModelInput create_model_input(core::DataTable &input_table, core::Country countr
 
     auto mapping = std::vector<MappingEntry>();
     for (auto &item : config.modelling.risk_factors) {
-        if (core::case_insensitive::equals(item.name, config.modelling.dynamic_risk_factor)) {
-            if (item.range.empty()) {
-                mapping.emplace_back(
-                    MappingEntry(item.name, item.level, core::Identifier{item.proxy}, true));
-            } else {
-                auto boundary = FactorRange{item.range[0], item.range[1]};
-                mapping.emplace_back(MappingEntry(item.name, item.level,
-                                                  core::Identifier{item.proxy}, boundary, true));
-            }
-        } else if (!item.range.empty()) {
+        if (item.range.empty()) {
+            mapping.emplace_back(MappingEntry{item.name, item.level, core::Identifier{item.proxy}});
+        } else {
             auto boundary = FactorRange{item.range[0], item.range[1]};
             mapping.emplace_back(
                 MappingEntry{item.name, item.level, core::Identifier{item.proxy}, boundary});
```

Comment:
I think this could be just:

```suggestion
        if (item.range.empty()) {
            mapping.emplace_back(item.name, item.level, item.proxy);
        } else {
            auto boundary = FactorRange{item.range[0], item.range[1]};
            mapping.emplace_back(item.name, item.level, item.proxy, boundary);
```